### PR TITLE
Provide selected DACs and datasets

### PR DESCRIPTION
**Pull request by vinjana**
Data is provided as JavaScript objects with datasets nested it DAC.datasets array.

Note that I added lodash as dependency. I got fed up of procedural-style iteration. It's small and  provides `_.cloneDeep(object)` and some functional-style API to collections.

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -26,19 +26,78 @@ let support = {
 
 };
 
-let studyTableView = {
+class StudyTable {
```
Comment (KochTobi) [inline js/data.js]: Ah very good. Makes it way more readable!

File: _includes/data/access.html [hunk 2]
```diff
@@ -50,9 +56,21 @@ <h2>Request Access</h2>
 			        	</div>
 			        </div>
 
-					<input type="button" name="sendButton" value="Send Mail" class="btn btn-default" onclick="test()">
+    					<input type="button" name="sendButton" value="Send Mail" class="btn btn-default" onclick="test()">
```
[the remark this answers is not in the document]
Comment (vinjana) [reply _includes/data/access.html]: Good question. I am used to 4 spaces. But I am also fine with 2. Problem here was that my editor at home (Atom) is different than the one I use at work (IDEA).

**Comment by vinjana** [comment]
I changed the indents to 4. The printing of contacts is also fixed. 
AFAICT this is in a useable state now and you (already) have approved. Do you want to have another look at the current state?

**Comment by vinjana** [comment]
Note that you can now access the website on the interims-URL [here](https://ghga.dkfz.de).

**Comment by KochTobi** [comment]
The interims URL lacks stylesheets for me though.
Probably not related to this PR

**Comment by vinjana** [comment]
Strange. I tested it with Vivaldi/Chrome and Firefox and everything is fine. Could you try figuring out what's the problem? Maybe on the developer console in the browser?

**Comment by KochTobi** [comment]
https://ghga.dkfz.de/ works now but the google maps is still in there for some reason. Should not be the case on the master branch.

**Pull request merged**